### ISSUE TITLE
Help getting started, examples seem out of date

### ISSUE THREAD

**Issue by sla-te** [derwentx/wp-api-python#15]
Tried the below snippet and it asks me to put a callback. (`TypeError: __init__() missing 1 required positional argument: 'callback'`)
```python
from wordpress import API

wpapi = API(
    url="http://example.com",
    consumer_key="XXXXXXXXXXXX",
    consumer_secret="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
    api="wp-json",
    version="wp/v2",
    wp_user="XXXX",
    wp_pass="XXXX",
    oauth1a_3leg=True,
    creds_store="~/.wc-api-creds.json"
)
```

**Comment by derwent-m-js** [comment]
Hey @chwba , could you post a full stack trace with line numbers? I don't believe this is from the WordPress API constructor.

**Comment by nioupola** [comment]
Hello,

You have to add the new argument to the API constructor:

```python
wpapi = API(
    url="https://example.com",
    consumer_key="XXXXXXXXXXXX",
    consumer_secret="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
    api="wp-json",
    version="wp/v2",
    wp_user="XXXX",
    wp_pass="XXXX",
    oauth1a_3leg=True,
    callback="https://example.com",
    creds_store="~/.wc-api-creds.json"
)
```

I forked it and fix the example and the encoding issue (#17): https://github.com/Synoptik-Labs/wp-api-python
